superset/views/base_api.py

```diff
@@ -107,13 +106,14 @@ def statsd_metrics(f: Callable[..., Any]) -> Callable[..., Any]:
     """
 
     def wraps(self: "BaseSupersetModelRestApi", *args: Any, **kwargs: Any) -> Response:
```

Comment:
This type can now be `BaseSupersetApi` as well (or anything that inherits from `BaseSupersetApiMixin`, not sure the best way to type-hint this though

Reply:
good point, changed it to `BaseSupersetApiMixin`